gspread/worksheet.py

```diff
@@ -3201,3 +3204,86 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        source: str,
+        condition_type: ValidationConditionType,
+        *values: Iterable[Any],
```

Comment:
I don't fancy having an unbounded list of arguments, it has been quiet complex to simplify this project and add typing.

I would prefer to have a single argument that hold either a list, a tuple etc (any iterable in fact) and keep the number of arguments fixed in size. 

@alifeee  what do you think about this ?

Reply:
I was going back and forth between unbouded values and an iterable. I have changed it in a35c39fbf80ae6b866f0eb38cda73c131666c4e1